bentoml/server/__init__.py

```diff
@@ -28,7 +34,21 @@ def start_dev_server(
     mb_max_latency: Optional[int] = None,
     run_with_ngrok: Optional[bool] = None,
     enable_swagger: Optional[bool] = None,
+    timeout: Optional[int] = None,
 ):
+    BentoMLContainer.bundle_path.set(bundle_path)
+
+    bento_server = BentoMLContainer.config.bento_server
+    bento_server.port.set(port or skip)
+    bento_server.timeout.set(timeout or skip)
+    bento_server.microbatch.timeout.set(timeout or skip)
+    bento_server.swagger.enabled.set(enable_swagger or skip)
+    bento_server.microbatch.max_batch_size.set(mb_max_batch_size or skip)
+    bento_server.microbatch.max_latency.set(mb_max_latency or skip)
```

Comment:
Should we move these to the CLI module closer to where they are defined? This function here instead takes `ModelApp` and `MarshalApp` as parameters. With DI, users should be able to create `ModelApp` and `MarshalApp` relatively easily. So this may be acceptable.

Reply:
No. `ModelApp` must be created after forked.